node/index.js

```diff
@@ -688,22 +693,6 @@ TChannelConnection.prototype.runInOp = function runInOp(handler, options, sendRe
     }
 };
 
-TChannelConnection.prototype._writeFrame = function _writeFrame(frame, callback) {
-    var self = this;
-    if (self.closing) {
-        self.logger.warn('write frame after close', {
```

Comment:
Really this check should be in `send()` and friends. `_writeFrame` probably got called synchronously anyway from `.send()`